src/_includes/backward-incompatible-changes/b2b/2.3.0-2.4.0-develop-mftf.html

```diff
@@ -0,0 +1,152 @@
+<h3 id="2.3.0-2.4.0-develop-page">Page changes</h3>
```

Comment:
shouldn't the b2b branches to be compared be `1.1-develop` an `1.2.0-develop`?

Reply:
should be `1.1.0-1.2.0-develop` to be consistent

Reply:
I was using 1.0.0 and 1.2.0-develop branches but I choose the wrong name for the file. I will rename it

Reply:
@dshevtsov when will you generate the rest of the BiC documents? I wonder if I should rerun it now or wait for later?

Reply:
I don't think it will change before the release. Final run will be right after the 2.4.0 release.